poppy/matrixDFT.py

```diff
@@ -195,14 +206,16 @@ def matrix_dft(plane, nlamD, npix,
     XU = np.outer(Xs, Us)
     YV = np.outer(Ys, Vs)
 
+    # SIGN CONVENTION: plus signs in exponent for basic forward propagation, with
+    # phase increasing with time. This convention differs from prior poppy version < 1.0
     if inverse:
-        expYV = np.exp(-2.0 * np.pi * -1j * YV).T
-        expXU = np.exp(-2.0 * np.pi * -1j * XU)
+        expYV = np.exp(-2.0 * np.pi * 1j * YV).T
+        expXU = np.exp(-2.0 * np.pi * 1j * XU)
         t1 = np.dot(expYV, plane)
         t2 = np.dot(t1, expXU)
     else:
-        expXU = np.exp(-2.0 * np.pi * 1j * XU)
-        expYV = np.exp(-2.0 * np.pi * 1j * YV).T
+        expXU = np.exp(-2.0 * np.pi * -1j * XU)
```

Comment:
Can you remove the double negative, and use the `two_pi_i` variable with the appropriate sign, like you did below?